torchtext/data/functional.py

```diff
@@ -1,19 +1,20 @@
 import re
+import io
+import logging
+from torchtext.utils import unicode_csv_reader
 
 __all__ = [
-    "generate_sp_model", "load_sp_model",
-    "sentencepiece_numericalizer", "sentencepiece_tokenizer",
-    "numericalize_tokens_from_iterator"
+    "generate_sp_model", "load_sp_model", "sentencepiece_numericalizer",
+    "sentencepiece_tokenizer", "numericalize_tokens_from_iterator"
 ]
-
-
 """
 This file contains experimental functionality.
 All of these are experimental, unstable, and subject to change or deletion.
 """
 
 
-def generate_sp_model(filename, vocab_size=20000,
+def generate_sp_model(filename,
```

Comment:
Are those changes relevant to the translation datasets? For good record, we avoid to fix multiple issues within a single PR.